core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -32,6 +32,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
+import org.apache.hadoop.fs.CanUnbuffer;
```

Comment:
in `core/client/fs` module, we shall not introduce a dependency to hadoop which will cause dependencies issues.
Module `core/client/hdfs` is the wrapper on top of Alluxio In/out streams and make them HDFS compatible.

Reply:
Got it, I have modified code for these issues